src/cli.nim

```diff
@@ -46,6 +46,18 @@ proc showVersion =
   echo &"Canonical Data Syncer v{NimblePkgVersion}"
   quit(0)
 
+proc showError(s: string) =
```

Comment:
I don't know if this idiomatic, but I would normally name the parameter `error` or `errorMessage` or something like that.